corehq/apps/es/users.py

```diff
@@ -150,7 +145,3 @@ def user_ids_at_locations_and_descendants(location_ids):
 
 def user_ids_at_locations(location_ids):
     return UserES().users_at_locations(location_ids).exclude_source().run().hits
-
```

Comment:
i am semi sure about this.
I think fetching locations without domain as in `user_ids_at_locations` would lead to more user_ids in the query than required for a particular domain if location_ids come straight from assigned_locations to a user which can span to different domains.
If this has already been considered when removing this method, great!